src/app/page/page.component.ts

```diff
@@ -126,6 +129,40 @@ export class PageComponent implements OnInit, OnDestroy {
     }
   }
 
+  setPageType(activePage): string {
```

Comment:
This now gets the page type instead of setting it.

Also, I think we can remove the `activePage` argument and reference `this.activePage` directly in the function body.

```suggestion
  getPageType(): string {
```